### ISSUE TITLE
Mod engine won't launch Elden Ring after updating with DLC

### ISSUE THREAD

**Issue by quanghuypham99** [soulsmods/ModEngine2#207]
Hello, as you know, Elden Ring released a new update on 20th June. After I update the game, the game don't launch when I execute 'launchmod_eldenring.bat'. Firstly, it opened a black window of Elden Ring, then the window terminated after a few seconds.

**Comment by ozrivera** [comment]
Having the same issue, I was using seamless-coop and now it won’t even load after I updated the game .

**Comment by Gillou68310** [comment]
Needs execute as admin

**Comment by kltsao** [comment]
Same issue.

**Comment by ozrivera** [comment]
> Needs execute as admin
Doesn’t work. The game won’t even launch.

**Comment by Gillou68310** [comment]
For me it's the opposite, the game won't launch when modengine2_launcher.exe is not run as admin

**Comment by misantronic** [comment]
I am having the same issue. fortunately, flawless widescreen etc. still does work

**Comment by kltsao** [comment]
Is this project still being updated?

**Comment by garyttierney** [comment]
> Is this project still being updated?

No. A separate project has been in the works to support Elden Ring post-DLC, and should be available sometime this weekend.

**Comment by BlueGunk** [comment]
Thanks Gary Tierney. You're probably aware the DMP file points to ModEngine.dll as being the faulting item. Look forward to the new project in due course.

**Comment by lackesis** [comment]
Same here, I think this is because "steam_appid.txt" no longer works for me. any idea?

**Comment by Satleixn** [comment]
Same problem. When I double-click bat file, nothing happened but CMD window flashed.

**Comment by Erebus1245** [comment]
Yeah this is not working anymore Elden Mod Loader still works but only loads dll mods not files.

**Comment by xuvvy** [comment]
Check your `config_eldenring.toml`, it is likely caused by an external DLL. In my case it was:

`eldenring_alt_saves.dll`

**Comment by BlueGunk** [comment]
Not in my case, xuvvy. As per my report it is the modengine2.dll faulting,
 which is pretty much the result of the latest game update.

**Comment by xuvvy** [comment]
> Not in my case, xuvvy. As per my report it is the modengine2.dll faulting, which is pretty much the result of the latest game update.

Gotcha. I can confirm that Mod Engine 2 works on my end without the specific DLL being loaded. I have, however, had this same issue in the past and what usually fixed it for me was verifying integrity of game files in Steam, then completely reinstalling ME2 and running it once before installing other mods.

**Comment by kane1117** [comment]
Everyone, please try updating ertransmogrify
I was fix by it

**Comment by Erebus1245** [comment]
I don't even use this mod it doesn't load at all even with no mods on a fresh install.

**Comment by Uklosk** [comment]
> > Is this project still being updated?
> 
> No. A separate project has been in the works to support Elden Ring post-DLC, and should be available sometime this weekend.

What is the new project name?

**Comment by lennylenlen** [comment]
I had no issues running mod engine, but the font and icons on the hud, inventory, and map are bugged. And the new DLC weapons appear in my character's hands like normal in game, but in the inventory or on hud, their icon is literally a big "ICON".

**Comment by Erebus1245** [comment]
> > > Is this project still being updated?
> > 
> > 
> > No. A separate project has been in the works to support Elden Ring post-DLC, and should be available sometime this weekend.
> 
> What is the new project name?


Most likely [this](https://github.com/garyttierney/me3) repo but it's not updated in months, i assume he's working on it privately.

**Comment by garyttierney** [comment]
> Most likely [this](https://github.com/garyttierney/me3) repo but it's not updated in months, i assume he's working on it privately.

Yeah, there's some final pieces to finish up and I'll be putting out an initial build tonight.

**Comment by peilong-du** [comment]
> > Most likely [this](https://github.com/garyttierney/me3) repo but it's not updated in months, i assume he's working on it privately.
> 
> Yeah, there's some final pieces to finish up and I'll be putting out an initial build tonight.

Is the build still coming out sometime soon ish? I can't get mod engine 2 to work no matter what I try.

**Comment by crucifux** [comment]
I'm using 1.12.2, and ModEngine2 loads with reg.bin stuff, as well as custom content from menu (ui), animations, and parts data, but nothing at all will load from /msg. dll mods load parallel with mod_loader like they used to. Because the only issue is menu/item text, I'm inclined to think this is From's way of blocking the big overhaul mods, or anything that 'reinterprets' their IP.

**Comment by hwayobi3** [comment]
> > 아마도 [이](https://github.com/garyttierney/me3) 저장소일 텐데 몇 달 동안 업데이트가 안 됐어요. 아마 비공개로 작업하고 있는 것 같아요.
> 
> 네, 마무리해야 할 몇 가지 최종 부분이 있고 오늘 밤에 초기 빌드를 내놓을 예정입니다.


주말에 새로운 버젼을 공개한다는 계획은 그대로인가요?
아니면 늦어지는건가요?

**Comment by Ryusennin** [comment]
No problem here either. I'm using a few mods like No Compass and Skin Replacer.

Make sure you disabled EAC and you started the Steam launcher before running the BAT file.

**Comment by hwayobi3** [comment]
> > Most likely [this](https://github.com/garyttierney/me3) repo but it's not updated in months, i assume he's working on it privately.

Is the plan to release a new version on the weekend in progress? Will it be delayed by any chance?

**Comment by Myoko** [comment]
modengine needs updated to be compatible with DLC.

**Comment by xuvvy** [comment]
> modengine needs updated to be compatible with DLC.

Untrue. Mod Engine 2 works with the latest Elden Ring version.

**Comment by Myoko** [comment]
> > modengine needs updated to be compatible with DLC.
> 
> Untrue. Mod Engine 2 works with the latest Elden Ring version.

Huh? If that's the case, it's very strange to me as it is incompatible with prayer and doesn't work with the armor MOD.

**Comment by Ryusennin** [comment]
ModEngine itself does work with Elden Ring 1.12.2. But some mods need to be updated (the text-based ones for example, like "Quest Log" or "Chat with Melina").

**Comment by crucifux** [comment]
> ModEngine itself does work with Elden Ring 1.12.2. But some mods need to be updated (the text-based ones for example, like "Quest Log" or "Chat with Melina").

I can get the failed read error ??text?? in game after making a mod right now, with 1.12.2 files. Its not an mod data issue, its an injection issue.

**Comment by xuvvy** [comment]
There might be issues with Mod Engine 2 that might need to be addressed, but it is the fact that it works. Works with the latest Erd Tools, Transmogify, Seamless Coop, Ultrawide UI Fixes.

People who are having issues should both check their ME2 ER config file and the 'mod' folder as having anything wrong there (e.g. old mods in the 'mod' folder or missing quotes on DLLs in the config) will most certainly make things break.

**Comment by Myoko** [comment]
I found that the armor mod doesn't work because ray tracing is enabled. disabling it does make the mod function properly, but the newly added Incantation from the DLC still have no effect. This issue needs to be resolved.

**Comment by lackesis** [comment]
I think the main issue is that the "SteamAppId=1245620" offline method no longer works for some people, as far as I know modengine2 also uses this method for offline. For some reason I was able to launch the game yesterday but not today.

**Comment by xuvvy** [comment]
> I found that the armor mod doesn't work because ray tracing is enabled. disabling it does make the mod function properly, but the newly added Incantation from the DLC still have no effect. This issue needs to be resolved.

Which incantations specifically have no effect and in which way? Is it only buffs?

I have used several DLC incantations and spells and they all work equally when launching the game through ME2 and vanilla.

**Comment by Myoko** [comment]
> > I found that the armor mod doesn't work because ray tracing is enabled. disabling it does make the mod function properly, but the newly added Incantation from the DLC still have no effect. This issue needs to be resolved.
> 
> Which incantations specifically have no effect and in which way? Is it only buffs?
> 
> I have used several DLC incantations and spells and they all work equally when launching the game through ME2 and vanilla.

erdtree's all the incantations have no effect for me. just like this
![ERD](https://github.com/soulsmods/ModEngine2/assets/3013386/a45ea98d-f0ef-4ac6-b7cf-f318db68b1f7)

**Comment by saedbysae** [comment]
does anyone have a solution for the bat file not opening ive tried EVERYTHING POSSIBLE

**Comment by lackesis** [comment]
> does anyone have a solution for the bat file not opening ive tried EVERYTHING POSSIBLE

Copy all files to the "Game" folder of Elden Ring, rename "modengine2_launcher.exe" to "start_protected_game.exe" (back up the old ofcs) and directly use "eldenring.exe" to start the game. This is how it works in my case since .bat doesn't work for me.

**Comment by SpokyOky** [comment]
> Copy all files to the "Game" folder of Elden Ring, rename "modengine2_launcher.exe" to "start_protected_game.exe" (back up the old one ofcs) and directly use "eldenring.exe" to start the game. This is how it works in my case since .bat doesn't work for me.

How do you load .toml config? And what does this config and your mods folder look like?

**Comment by BlueGunk** [comment]
Thanks all. Yes, my Mod Engine is now working thanks to the above comments. One of my mods is causing a problem.

**Comment by DANTELEAS** [comment]
New DLC items now do not load icons for all DLC items if you launch the game through Modengine 2. HOW TO FIX ICONS THANKS.

**Comment by k1do101** [comment]
for some reason when i install the mod engine it doesn't come with the .bat file so i can't open the game, does anyone know what to do?
